packages/react/src/RelativeTime/RelativeTime.tsx

```diff
@@ -1,8 +1,18 @@
+import React from 'react'
 import {RelativeTimeElement} from '@github/relative-time-element'
 import type {ComponentProps} from '../utils/types'
 import {createComponent} from '../utils/custom-element'
 
-const RelativeTime = createComponent(RelativeTimeElement, 'relative-time')
+const RelativeTimeComponent = createComponent(RelativeTimeElement, 'relative-time')
 
-export type RelativeTimeProps = ComponentProps<typeof RelativeTime>
+const localeOptions: Intl.DateTimeFormatOptions = {month: 'short', day: 'numeric', year: 'numeric'}
+function RelativeTime({date, ...props}: RelativeTimeProps) {
+  return (
+    <RelativeTimeComponent {...props} date={date}>
```

Comment:
When I pass `datetime` instead of `date`, like `<RelativeTime datetime="2024-03-10T16:30:00-08:00" />` in a sample app, it doesn't render any date in SSR. Seems like the props return undefined. Is there anything I am missing here? 🤔

Reply:
I've fixed this and added tests to confirm.